### DISCUSSION TITLE
Welcome to icy_term Discussions!

### DISCUSSION THREAD

**Discussion by mkrueger** [mkrueger/icy_term#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by n4tivez** [comment]
Hey,

Looking good. Closely watching the progress on iCYterm.

-- n4tive

**Reply by mkrueger** [reply]
Just let me know if I do crap :)

**Reply by n4tivez** [reply]
Will do.

Love the newly added shader.

**Reply by mkrueger** [reply]
ty - next version will have settings to turn off & on smoothing, effects etc. 
Personally I prefer more a sharper, more blocky style. However it looks great with PETSCII. It feels like I have my old C64 in front of me.
